src/Files.App/UserControls/Widgets/DrivesWidget.xaml

```diff
@@ -8,12 +8,34 @@
 	xmlns:local="using:Files.App.UserControls.Widgets"
 	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 	mc:Ignorable="d">
-	<Grid>
+
+	<local:HomePageWidget.Resources>
```

Comment:
Is this out of scope?

Reply:
I dont think so. This card shound not have the same padding & style as the default one. On hovering they will be emphasized like settings control in win11 settings.

Reply:
So it is changed margin to 12,8 or so, shouldn't be? @mdtauk

Reply:
What's the before and after?

Reply:
![image](https://github.com/files-community/Files/assets/7389110/8d6a0b06-9e65-47c3-9dbf-9ebda21de1e2)

Reply:
Before
![image](https://github.com/files-community/Files/assets/62196528/adfeb8b8-074f-429c-b12e-c9d36c5e04ca)

After
![image](https://github.com/files-community/Files/assets/62196528/9484b11d-e987-4c79-9a51-08485a97d5a1)

Reply:
What about emphasizing border styling (behaviour is like settings button in win11 settings app)? should be removed?

Reply:
We haven't made any decisions on the card style so this should be limited to updating the expander style.

Reply:
All removed!